src/python_testing/TC_RVCRUNM_2_2.py

```diff
@@ -154,7 +154,8 @@ async def test_TC_RVCRUNM_2_2(self):
 
         if self.mode_a not in self.supported_run_modes_dut or \
                 self.mode_b not in self.supported_run_modes_dut:
-            asserts.fail("PIXIT.RVCRUNM.MODE_A and PIXIT.RVCRUNM.MODE_B must be valid supported modes.")
+            asserts.fail(
+                f"PIXIT.RVCRUNM.MODE_A and PIXIT.RVCRUNM.MODE_B must be valid supported modes. Valid modes: {self.supported_run_modes_dut}")
```

Comment:
Can we include the values of self.mode_a and self.mode_b here too?

Reply:
sure.